data/yaml/venues.yaml

```diff
@@ -8,10 +8,20 @@ acl:
   is_toplevel: true
   name: Annual Meeting of the Association for Computational Linguistics
   oldstyle_letter: P
+aespen:
+  acronym: AESPEN
+  is_acl: false
```

Comment:
`is_acl` will default to False if it's not given, so if we want to reduce clutter in this file (especially now with more and more venues being added), we could remove these lines.

Reply:
Great, thanks!